services/post_service.py

```
@@ -40,9 +40,10 @@ def __init__(self, content_dir, use_cache=True):
         self.post_dir = self.content_dir / "post"
         self.use_cache = use_cache
 
-        # 初始化缓存服务
+        # 初始化缓存服务，每个内容目录使用独立的缓存数据库
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)